docs/SETUP.md

```diff
@@ -30,6 +30,12 @@ This project uses the following third party library dependencies:
 | libzip          | 1.7.3                    |
 | libcgroup-tools | 2.0                      |
 
+## Users/Groups
+
+Greengrass nucleus lite will use the users/groups configured in the
+[build process](BUILD.md#usersgroups). If you're using the apt package, yocto
+layer, or yocto image, these will be set up and configured for you.
```

Comment:
I would also mention what the ones in the samples are so they can set it below without needing to go through the build guide. Or maybe just mention what posixUser setting to use for demo images/apt package.

Reply:
Added below in the posixUser section